org.geany.Geany.appdata.xml

```diff
@@ -40,7 +40,7 @@ as possible from a specific Desktop Environment like KDE or GNOME.</p>
   <update_contact>https://github.com/geany/geany</update_contact>
   <translation type="gettext">geany</translation>
   <releases>
-    <release date="2018-02-25" version="1.33"/>
+    <release date="2019-04-28" version="1.35"/>
```

Comment:
don't replace the release tag, append a new one on top instead
```suggestion
    <release date="2019-04-28" version="1.35"/>
    <release date="2018-02-25" version="1.33"/> 
```

Reply:
Agreed. I'll merge this and add in a separate commit though to get this in.